wellcomeml/ml/cnn.py

```diff
@@ -233,11 +240,17 @@ def residual_attention(x1):
         output_activation = (
             "sigmoid" if nb_outputs == 1 or self.multilabel else "softmax"
         )
+        if self.feature_approach == "multilabel-attention":
+            # each output comes from an attention head
+            nb_outputs = 1
```

Comment:
It's a bit weird that you use nb_outputs as it's actual value for the output activation, then you change it then use it again, but that's just style

Reply:
I understand this is confusing, will try to improve terminology in the future